scripts/dev-e2e.sh

```diff
@@ -123,11 +141,10 @@ main() {
   fi
   echo "Checking for armadactl .."
   if ! test -x "$scripts"/armadactl; then
-    log "Fetching armadactl..."
-    fetch-armadactl
+    fetch-armadactl | log_group "Fetching armadactl"
```

Comment:
maybe all these pipes to log_group should include std error, just so people don't have to think about it.